.vsts-ci.yml

```diff
@@ -129,11 +129,13 @@ jobs:
         TASK: bdist
         PYTHON_VERSION: 3.5
   steps:
-  - powershell: Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
-    displayName: Enable conda
+  - powershell: |
+      Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
+      Write-Host "##vso[task.setvariable variable=AZURE]true"
+    displayName: 'Set Variables'
   - script: |
-      cmd /c 'activate & conda config --set always_yes yes --set changeps1 no & conda update -q -y conda & conda create -q -y -n %CONDA_ENV% python=%PYTHON_VERSION% joblib matplotlib numpy pandas psutil pytest python-graphviz "scikit-learn<=0.21.3" scipy'
-      cmd /c "activate %CONDA_ENV% & powershell -ExecutionPolicy Bypass -File %BUILD_SOURCESDIRECTORY%/.ci/test_windows.ps1"
+      cmd /c "conda init powershell"
```

Comment:
I think we do `init` in `test_windows.ps1#16` and this line is redundant.

Reply:
I intentionally did it twice, actually.

When you run this command it says:

```
For changes to take effect, close and re-open your current shell. 
```

And I was getting errors like "`pytest` is not a recognized cmdlet..." that told me that `conda activate`  wasn't working.

I can push a commit testing without it though! Maybe those errors were caused by something else I fixed while testing stuff.

Reply:
As of https://github.com/microsoft/LightGBM/pull/2962/commits/64ef6ac6aeac47d443b9079547dd58d6091c8b5c I see the issue:

![image](https://user-images.githubusercontent.com/7608904/78100690-0513b180-73ab-11ea-80d0-a7955aca671a.png)

Once I put back in the duplicate `conda init powershell`, it works again (https://github.com/microsoft/LightGBM/pull/2962/commits/109e152cd5dea022e0b769f9fc900a5d7f3434c3)

Reply:
Ah, I see! Then I think it's worth commenting this line to prevent its removal in the future.

Reply:
Never mind! I see you added this line into both yaml configurations, so it should be quite obvious.